go.mod

```diff
@@ -10,6 +10,7 @@ require (
 	github.com/onsi/gomega v1.27.10
 	github.com/spf13/cobra v1.7.0
 	github.com/spf13/pflag v1.0.5
+	go.uber.org/mock v0.2.0
```

Comment:
This should be hopefully dropped after the comment about not passing manager.Manager to the validator.